src/csharp/Grpc.Core.Api/SerializationContext.cs

```diff
@@ -39,11 +39,22 @@ public virtual void Complete(byte[] payload)
 
         /// <summary>
         /// Gets buffer writer that can be used to write the serialized data. Once serialization is finished,
-        /// <c>Complete()</c> needs to be called.
+        /// <c>Complete()</c> needs to be called. A <c>null</c> value will be returned if serialization
+        /// with a buffer writer is not supported.
         /// </summary>
         public virtual IBufferWriter<byte> GetBufferWriter()
         {
-            throw new NotImplementedException();
+            return null;
```

Comment:
sg, but depends on the outcome of https://github.com/grpc/grpc/issues/20688#issuecomment-544844583

Reply:
Undone. Can wait until approach is figured out.